test/socket_client_test.exs

```diff
@@ -60,21 +60,31 @@ defmodule Phoenix.Channels.GenSocketClientTest do
     assert_receive {TestSite.Channel, {:terminate, {:shutdown, :left}}}
   end
 
-  test "client message push references" do
+  test "push references on the same channel" do
```

Comment:
Hm... what we are testing here is really that the references are unique? This test name doesn't tell us much. Maybe `push references are unique by being monotonically increasing`